lighthouse-core/lib/emulation.js

```diff
@@ -113,9 +120,19 @@ function goOffline(driver) {
   return driver.sendCommand('Network.emulateNetworkConditions', OFFLINE_METRICS);
 }
 
+function enableCPUThrottling(driver) {
+  return driver.sendCommand('Emulation.setCPUThrottlingRate', CPU_THROTTLE_METRICS);
+}
+
+function disableCPUThrottling(driver) {
+  return driver.sendCommand('Emulation.setCPUThrottlingRate', NO_CPU_THROTTLE_METRICS);
+}
+
```

Comment:
Just a thought: I'm curious if it's possible for us to get clever with Lighthouse's usage of CPU throttling. As throttling is relative to the system you're testing on, you can experience some pretty wildly different results. 5x throttling on a high-end Mac is going to be different to 5x throttling on a lower end computer. I wonder if it might be possible to do a pass that could determine system "highendness" and normalize throttling accordingly.